mcs/class/System.Net.Http/Makefile

```diff
@@ -10,6 +10,9 @@ LIB_MCS_FLAGS = $(EXTRA_LIB_MCS_FLAGS)
 ifeq (monodroid,$(PROFILE))
 LIB_MCS_FLAGS += -d:XAMARIN_MODERN
 endif
+ifeq (wasm,$(PROFILE))
```

Comment:
This should not be needed, we add `-d:WASM` in `mcs/build/profiles/wasm.make`.

Reply:
Indeed it does work.